### PR TITLE
PMM-6567 pg_stat_monitor

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

**Comment by BupycHuk** [comment]
could you include links to other PRs and FB, please?

**Comment by JiriCtvrtka** [comment]
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1104

PRs:
PMM - https://github.com/percona/pmm/pull/584
PMM managed - https://github.com/percona/pmm-managed/pull/457
PMM API tests - https://github.com/Percona-Lab/pmm-api-tests/pull/124
PMM agent - https://github.com/percona/pmm-agent/pull/186

File: api/qanpb/collector.proto [hunk 1]
```diff
@@ -335,6 +335,12 @@ message MetricsBucket {
   float m_blk_write_time_cnt = 222;
   // Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
   float m_blk_write_time_sum = 223;
+  float m_cpu_user_time_cnt = 230;
+  // TODO:
```
Comment (askomorokhov) [inline api/qanpb/collector.proto:339]: Please add a comment here.

File: api/qanpb/collector.proto [hunk 2]
```diff
@@ -335,6 +335,12 @@ message MetricsBucket {
...
+  float m_cpu_user_time_sum = 231;
+  float m_cpu_sys_time_cnt = 232;
+  // TODO:
```
Comment (askomorokhov) [inline api/qanpb/collector.proto:342]: and here a comment as well.

**Review by askomorokhov — APPROVED** [review]
Looks good to me with comments.

**Pull request merged**